docs/modules/ROOT/pages/tutorial-kubernetes.adoc

```diff
@@ -216,150 +150,401 @@ Specifically we use the `kubernetes_sd_config` to automatically discover Couchba
-image::prometheus-successful-targets.png[]
+[NOTE]
+Remove the check from “Add to Prometheus” option, because prometheus scraping will be configured using service discovery.
+
+.Add cluster image
+image::add-cluster-k8s.png[]
+
+As soon as you add a cluster, you will see a Grafana URL where you can view inventory and metrics of Couchbase server clusters.
+
+.Couchbase inventory image
+image::couchbase-inventory-k8s.png[]
+
+== Prometheus Targets
+
+From the "Prometheus Targets" option, we can see the prometheus targets and their details.
+For instance, we can filter the targets to show all targets or unhealthy targets.
+The state information tells which prometheus targets are running.
+The last scrape value shows how long ago the target metrics were scraped.
+
+.Prometheus target image
+image::prometheus-target-k8s.png[]
 
 Whilst the pods are coming up, some may report as failing but these will resolve once the pods are running.
 
-.Failing targets during startup
-image::prometheus-failing-targets.png[]
+.Prometheus target failing image
+image::prometheus-target-failing-k8s.png[]
 
-=== CMOS cluster configuration
+== Grafana Dashboard
 
-[NOTE]
-====
-Unfortunately cluster monitor is currently designed for an on-premise solution so will not cope with auto-discovery on Kubernetes yet.
-Without this the various Grafana dashboards will not function correctly so you have to do it manually for now.
-====
+With Grafana, multiple dashboards for monitoring a Couchbase cluster are provided out of the box.
+You can list all the dashboards using the https://grafana.com/docs/grafana/latest/dashboards/search/#dashboard-search[search dashboard option^].
+You may create https://grafana.com/docs/grafana/latest/getting-started/getting-started/#step-3-create-a-dashboard[additional dashboards^] as per your needs. 
+The following are some out-of-the-box dashboards.
+
+=== Couchbase Cluster Overview Metrics 
+
+Couchbase cluster overview metrics dashboard can be accessible on Grafana with name as: `single-cluster-overview`.
+This dashboard displays a number of items including Couchbase nodes, buckets available, version information, health check warnings, and which services are running.
+
+.Couchbase cluster overview image
+image::couchbase-cluster-overview-k8s.png[]
+
+=== Couchbase Node Overview Metrics
+
+Couchbase node overview metrics dashboard can be accessible on Grafana with name as: `node-overview`.
+This dashboard displays from a node perspective, resource utilization, version, and health check warnings.
+
+.Couchbase node overview image
+image::couchbase-node-overview-k8s.png[]
+
+== Alerts
+
+CMOS comes with pre-installed alert rules to monitor the Couchbase cluster.
+Navigate to Alertmanager, prometheus UI to check the alert rules and alerts.
+For more information check the prometheus and alerting configuration section.
+
+.Alert rules image
+image::prometheus-alert-rules.png[]
+
+.Alerts image
+image::prometheus-alerts.png[]
+
+== Alert Manager
```

Comment:
Is it `Alert Manager` or `Alertmanager `? Bit confused...
```suggestion
== Alertmanager
```

Reply:
I think it should be Alertmanager, that's the official spelling.